tests/infrared/16.1/infrared-openstack.sh

```diff
@@ -1,116 +0,0 @@
-#!/usr/bin/env bash
-set -e
-
```

Comment:
Not sure where this is coming from. There is no 16.1 dir(?)

Reply:
In your PR there is. This PR is against your PR.

Reply:
Ugh, I missed this is a PR against my PR. The UI utterly sucks here.